examples/concertbot/config.yml

```diff
@@ -6,3 +6,6 @@ policies:
   - name: FallbackPolicy
   - name: MemoizationPolicy
   - name: MappingPolicy
+
+language: en
+pipeline: supervised_embeddings
```

Comment:
It is needed to execute some of the `rasa` cli commands. If that is not really intuitive, we might want to open a ticket for fixing this. But for now, this needs to be here.

Reply:
Hmm this is for interactive learning right? Yes, I think we should open a ticket - you shouldn't have to pass any policies, it would be the same thing as doing interactive learning with the old command and just not passing a `--nlu` model. Any other commands this is needed for?

Reply:
Yes, only needed for interactive learning. Just double checked. Will open a ticket.